packages/atomic-angular/README.md

```diff
@@ -1,27 +1,142 @@
-# AtomicAngular
+# Atomic Angular
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.1.4.
+An Angular component library for building modern UIs interfacing with the Coveo platform. Atomic Angular is a wrapper around the core [Atomic](https://docs.coveo.com/en/atomic/latest/) web component library.
 
-## Development server
+The integration of Angular-based projects with Web based components can be tricky. This project is meant to address this issue, making it possible to use Atomic web-components in a manner that feels more natural to developers familiar with Angular.
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+## Installation
 
-## Code scaffolding
+`npm i @coveo/atomic-angular`
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+## Usage
 
-## Build
+Since Atomic Angular is built on top of the core [Atomic](https://docs.coveo.com/en/atomic/latest/) web-components library, the vast majority of concepts that apply to core Atomic will apply to Atomic Angular.
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
+However, there are still some special considerations.
 
-## Running unit tests
+## Static Assets - Languages and SVGs
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+For performance reasons, the generated JavaScript bundle does not automatically include static assets that are loaded on demand. This impacts language support, as well as the use of included SVG icons.
 
-## Running end-to-end tests
+It is mandatory that you make available external assets distributed with Atomic Angular by including them in the public directory of your app. Without this, for example, labels in the app will appear as temporary placeholders.
 
-Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+The location of the public directory depends on how you build, configure and distribute your app.
 
-## Further help
+For example, for any project created with [Angular CLI](https://angular.io/cli), this would mean copying language and icon assets to the root of the source directory
 
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+```
+cp -r node_modules/@coveo/atomic-angular/assets src/assets src/assets
+cp -r node_modules/@coveo/atomic-angular/lang src/lang
+```
+
+It is important to respect the folder hierarchy, with SVG icons under the `assets` subdirectory, and labels and languages under the `lang` subdirectory.
```

Comment:
I've added this section and the `Including default Coveo Theme` section since I had forgotten to talk about those concepts @jpmarceau